### PR TITLE
Check-in Management & Ticket QR Code

### LINKED ISSUE #2325: Implement Check-in Management in Attendees List and offer Access through an API
In order to enable organizers to check in and view checked in attendees, we need check-in management. The attendee check-in is a tool which allows organizers to manage attendees check-in at the event day. It can be added to the Attendees List.

Required features:
* [x] Organizers should be able to find or verify registered attendees by name, email, order no. or ticket code in `events/ID/tickets/attendees/`

screenshot from 2016-08-20 16 40 05

* [x] Implement a button in the last column "Check In" or "Undo"

screenshot from 2016-08-20 16 42 00

Also organizers should be able:
* [x] Access and confirm attendees through QR code that can be scanned with an Android phone and accesses the Orga Server through an API
   * [x] Therefore implement the addition of a QR code to every ticket
   * [x] Define a verified API access e.g. that can be used by an Android phone. This API access would require authentication with a user account identifying the user as an organizer, co-organizer, moderator or other event role

### PR DESCRIPTION
**Pull request by niranjan94**
Resolves #2325

### CODE REVIEW

**Comment by SaptakS** [comment]
Please fix the travis build.

**Comment by niranjan94** [comment]
@SaptakS this is not ready for merge. There will be failing builds ;)

File: app/api/__init__.py [hunk 1]
```diff
@@ -42,6 +43,9 @@
 api.add_namespace(extras_api)
 api.add_namespace(notifications_api)
 api.add_namespace(error_models)
+api.add_namespace(error_models)
```
Comment (aviaryan) [inline app/api/__init__.py]: Is `api.add_namespace(error_models)` duplicated here ?

**Comment by icedwater** [comment]
@niranjan94 can I take this to mean that the QR code feature is still in alpha? I'd like to know where I can open issues related to it.

**Comment by niranjan94** [comment]
@icedwater if you face issues scanning the QR via the organiser app or other organiser app related bugs, you can open an issue at https://github.com/fossasia/open-event-orga-app/issues. 

For anything else you can open an issue at https://github.com/fossasia/open-event-orga-server/issues

**Comment by icedwater** [comment]
Thanks @niranjan94 for the quick response, and for actually working on the QR code feature in the first place. I will post an issue here.

**Pull request merged**